arm/Microsoft.Storage/storageAccounts/.parameters/encr.parameters.json

```
@@ -0,0 +1,38 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>azsaencr001"
+        },
+        "storageAccountSku": {
+            "value": "Standard_LRS"
+        },
+        "allowBlobPublicAccess": {
+            "value": false
+        },
+        "publicNetworkAccess": {
+            "value": "Disabled"
+        },
+        "requireInfrastructureEncryption": {
+            "value": true
+        },
+        "systemAssignedIdentity": {
+            "value": false
+        },
+        "userAssignedIdentities": {
+            "value": {
+                "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001": {}
+            }
+        },
+        "cMKKeyVaultResourceId": {
+            "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.KeyVault/vaults/adp-<<namePrefix>>-az-kv-nopr-002"
+        },
+        "cMKeyName": {
```

Just looking at the other parameters, should this be "prefixed" `cMK` as well?
```suggestion
        "cMKKeyName": {
```

---

`CMK` = The mode of managing encryption keys.
`Key` is an "object"/"class" of the mode
`Name` is a property of the key.
= `CMKKeyMode`

---

done